tests/pipeline-all-amps/repo/Dockerfile

```diff
@@ -45,6 +45,18 @@ logger.alfresco-repo-content-transform-TransformerDebug.level=debug\n\
 # Grant all security permissions to jolokia and share in order to work properly.
 RUN sed -i -e "\$a\grant\ codeBase\ \"file:\$\{catalina.base\}\/webapps\/jolokia\/-\" \{\n\    permission\ java.security.AllPermission\;\n\};\ngrant\ codeBase\ \"file:\$\{catalina.base\}\/webapps\/share\/-\" \{\n\    permission\ java.security.AllPermission\;\n\};" ${TOMCAT_DIR}/conf/catalina.policy
 
+# Switch between currently installed JRE version and 11
```

Comment:
Should be `17`

Reply:
`21`, since `17` is currently installed.